addons/binding/org.openhab.binding.icloud/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,30 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: iCloud Binding
+Bundle-SymbolicName: org.openhab.binding.icloud;singleton:=true
+Bundle-Vendor: openHAB
+Bundle-Version: 2.2.0.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-ClassPath: .
+Import-Package: com.google.gson;resolution:=optional,

```

Comment:
Why would the import of `com.google.gson` be optional, if it misses it should be installed by the framework, so imho the optional should be left out here.